### PR TITLE
Update readme with section detailing install

### PR DESCRIPTION
**Pull request by jimson-msft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,24 @@ With the sdk installed, you can now build the plugin by navigating back to the p
 
 At this point, you should have built and packaged all components
 
+### Installing DO Client components
+
+There are a couple ways for you to install the DO client components
+
+1. If you have built the component into a debian package, you can simply find the debian package and install like detailed above
```
Comment (shishirb-MSFT) [inline README.md:95]: >   [](http://example.com/codeflow?start=128&length=2)

End of sentence. Place a period (.) ?

File: README.md [hunk 2]
```diff
@@ -88,6 +88,24 @@ With the sdk installed, you can now build the plugin by navigating back to the p
...
+> cd /tmp/build-deliveryoptimization-sdk/linux-debug/
+> sudo apt-get install ./libdeliveryoptimization*.deb
+> cd /tmp/build-deliveryoptimization-agent/linux-debug/
+> sudo apt-get install ./deliveryoptimization-agent*.deb
+> cd /tmp/build-deliveryoptimization-plugin-apt/linux-debug/
+> sudo apt get install ./deliveryoptimization-plugin-apt*.deb
+```
+
+2. If you build and install using cmake, or through some other custom means, be sure to setup the DO user/groups correctly in your installation  
+You can reference this [script](https://github.com/microsoft/do-client/tree/main/client-lite/build) as a reference
```
Comment (shishirb-MSFT) [inline README.md:108]: >script [](http://example.com/codeflow?start=24&length=6)

Did you mean to link to the postinst script? Right now it takes user to the parent folder.

File: README.md (hunk 1 above)
Comment (shishirb-MSFT) [reply README.md:95]: Next line as well.

File: README.md (hunk 2 above)
Comment (jimson-msft) [reply README.md:108]: Yep, thanks

**Review by shishirb-MSFT — APPROVED** [review]
:shipit:

**Pull request merged**